py/tests/modeset_event.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/python3
+
+import pykms
+import selectors
+import sys
+
+def readdrm(fileobj, mask):
+    for ev in card.read_events():
+        ev.data(ev)
+
+def waitevent(sel):
+    events = sel.select(1)
+    if not events:
+        print("Error: timeout receiving event")
+    else:
+        for key, mask in events:
+            callback = key.data
+            callback(key.fileobj, mask)
+
+def eventhandler(event):
+    print("Received %s event successfully (seq %d time %f)" %
+          (event.type, event.seq, event.time))
+
+card = pykms.Card()
+sel = selectors.DefaultSelector()
+sel.register(card.fd, selectors.EVENT_READ, readdrm)
+
```

Comment:
Not saying anything should be changed, but just pointing out that "readdrm" here is just plain data. So you don't _need_ to pass a function here, you could just have the "waitevent" function and do everything in that, without needing a separate "readdrm" function.